static/js/apps/eval_rig/eval_list.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Component to show the full list of evaluation queries */
+
+import { User } from "firebase/auth";
+import React, { useState } from "react";
+import { Button, Input, Modal } from "reactstrap";
+
+import { getCallExistence } from "./data_store";
+import { DcCall } from "./eval_section";
+import { Query } from "./query_section";
+
+interface EvalListPropType {
+  user: User;
+  sheetId: string;
+  queries: Record<string, Query>;
+  calls: Record<string, DcCall[]>;
+  onQuerySelected: (query: Query) => void;
+}
+
+export function EvalList(props: EvalListPropType): JSX.Element {
+  const [userEvalsOnly, setUserEvalsOnly] = useState(false);
+  const [modalOpen, setModalOpen] = useState(false);
+  const [queryCompletionStatus, setQueryCompletionStatus] = useState({});
+
+  const userLdap = props.user.email.split("@")[0];
```

Comment:
i wonder if we should use full email as user id since we want this to be used externally or by 3P raters.